.readme-partial.json

```diff
@@ -0,0 +1,4 @@
+[
+    "### **_IMPORTANT NOTE_**",
+    "**This repository is to be considered _DEPRECATED_ and version 1.2.0 `dialogflow` was moved towards v2.0.0 `@google-cloud/dialogflow`.**"
```

Comment:
Woah - I don't think the *repository* is deprecated.  The `dialogflow` module has just been renamed to `@google-cloud/dialogflow`.  I want to be careful we don't overstate this :)

Reply:
how about putting `This repository is _RENAMED_ from version 1.2.0 `dialogflow` towards v2.0.0 `@google-cloud/dialogflow`.` ?

Reply:
The issue is the *repository* wasn't named. It was the *package*.  But otherwise, yes :)

Reply:
changed the message to `This package is RENAMED from version 1.2.0 dialogflow towards v2.0.0 @google-cloud/dialogflow.`